<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPanelAffineTransforms</class>
 <widget class="QDialog" name="LPanelAffineTransforms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>264</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Affine transformations</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>176</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mLayoutVertical">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="mLabelTitle">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Affine transforms&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mLabelRotation">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Rotation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="mSliderRotation">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mLabelTranslation">
      <property name="text">
       <string>Translation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="mSliderTranslationX">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="mSliderTranslationY">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mLabelScale">
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="mSliderScale">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
